guardian/recipes-backend
# Connect to facia

## What does this change?

Sets up recipes-backend to be able to receive content sent from Fronts Tool (aka Facia) as per https://github.com/guardian/facia-tool/pull/1565

## How to test

- Deploy to CODE
- Ensure that the SQS queue created by CDK here is subscribed to the Feast output topic in the CMS Fronts account
- Ensure that the latest version of https://github.com/guardian/facia-tool/pull/1565 is running on facia
- Publish Feast curation from facia CODE
- Watch the new lambda receive it
- It should be available at the usual endpoints

## How can we measure success?

Able to publish Feast curation to CODE from Facia

## Have we considered potential risks?

- At present, this will over-write curation data already in place. Need to look at protecting mise-en-place content from overwriting.
